addons/portal/controllers/portal.py

```diff
@@ -420,11 +420,11 @@ def details_form_validate(self, data, partner_creation=False):
 
     def _get_mandatory_fields(self):
         """ This method is there so that we can override the mandatory fields """
-        return self.MANDATORY_BILLING_FIELDS
+        return ["name", "phone", "email", "street", "city", "country_id"]
```

Comment:
Actually maybe we can `list(MANDATORY_BILLING_FIELDS)` then, in case somebody was overwriting it for some reason

Since we're adding master TODOs anyway